loadclient/query.go

```diff
@@ -0,0 +1,25 @@
+package loadclient
+
+import (
+	"fmt"
+)
+
+func QueryLog(queries []string,options Options) {
+	opt = options
+	if opt.DestinationAPIURL != "" {
+		switch opt.Destination {
+		case "loki":
+			fmt.Println("Query to loki: TO BE IMPLEMENTED")
+		case "elasticsearch":
```

Comment:
The cases are an enumeration, it should be a good choice to make this visible on the `Options` struct for the user.

Reply:
Fixed
DONE